### PR TITLE
Adding ruls

### PR DESCRIPTION
**Pull request by Chiamakabrowneyes**
# Pull Request Template

## Description

Added a backend feature to edit and update the rules part of the application

## Type of change

- [X ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested / How Can This Be Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
By viewing the url --> app/rules

image
image

### CODE REVIEW

**Comment by bathientran** [comment]
Nice – good stuff taking a stab here.

There are a couple of issues though:
- It doesn't seem to be optimized for mobile 
image
- The rules aren't accessible in any way at the moment in the frontend (except navigating to `/rules` manually)
- There is no backend for this

Can you work with someone from the backend team to build out this feature?

File: src/app/api/announcements/route.js
Comment (bathientran) [inline src/app/api/announcements/route.js:1]: You have to remove any of this code – it breaks our app.

File: src/app/contactMe/contactMe.css
Comment (bathientran) [inline src/app/contactMe/contactMe.css:1]: Is this part of the rules feature?

File: src/app/_components/CreateEntityBox/CreateEntityBox.js
Comment (bathientran) [inline src/app/_components/CreateEntityBox/CreateEntityBox.js:1]: You should remove any of these

**Review by bathientran — CHANGES_REQUESTED** [review]

File: src/app/api/announcements/route.js
Comment (Chiamakabrowneyes) [reply src/app/api/announcements/route.js:1]: Thanks for the feedback Ba Thien. I removed the dead code, and worked on the app to make it adaptable to mobile. Heres what it looks like. I'm trying to create a commit to this PR, but I dont know why the push isnt visible in this PR
image

File: src/app/contactMe/contactMe.css
Comment (Chiamakabrowneyes) [reply src/app/contactMe/contactMe.css:1]: Nope, removed!

File: src/app/_components/CreateEntityBox/CreateEntityBox.js
Comment (Chiamakabrowneyes) [reply src/app/_components/CreateEntityBox/CreateEntityBox.js:1]: Removed

**Comment by Chiamakabrowneyes** [comment]
@bathientran @vickipetrova 
I was trying to push the new changes that I made, but accidentally shared it to this PR: https://github.com/minerva-startup-spring2024/roommate-organizer/pull/62 . The deployment is successful here

**Pull request merged**